schema.json

```diff
@@ -0,0 +1,2729 @@
+                                    "type": "boolean",
+                                    "description": "Reverse the order of the underlying list.",
+                                    "default": false
+                                },
+                                "sortKey": {
+                                    "@ref": "takeshapetestsite:ProductVariantSortKeys",
+                                    "description": "Sort the underlying list by the given key.",
+                                    "default": "POSITION"
+                                }
+                            }
+                        },
+                        "description": "A list of variants associated with the product."
+                    },
+                    "vendor": {
+                        "type": "string",
+                        "@mapping": "takeshapetestsite:Product.vendor",
+                        "description": "The name of the product's vendor."
+                    }
+                },
+                "required": [
+                    "availablePublicationCount",
+                    "collections",
+                    "createdAt",
+                    "defaultCursor",
+                    "description",
+                    "descriptionHtml",
+                    "descriptionPlainSummary",
+                    "handle",
+                    "hasOnlyDefaultVariant",
+                    "hasOutOfStockVariants",
+                    "id",
+                    "images",
+                    "inCollection",
+                    "isGiftCard",
+                    "legacyResourceId",
+                    "media",
+                    "mediaCount",
+                    "metafields",
+                    "options",
+                    "priceRange",
+                    "priceRangeV2",
+                    "privateMetafields",
+                    "productPublications",
+                    "productType",
+                    "publicationCount",
+                    "publications",
+                    "publishedOnChannel",
+                    "publishedOnCurrentChannel",
+                    "publishedOnCurrentPublication",
+                    "publishedOnPublication",
+                    "requiresSellingPlan",
+                    "resourcePublications",
+                    "resourcePublicationsV2",
+                    "sellingPlanGroupCount",
+                    "seo",
+                    "status",
+                    "storefrontId",
+                    "tags",
+                    "title",
+                    "totalInventory",
+                    "totalVariants",
+                    "tracksInventory",
+                    "translations",
+                    "unpublishedChannels",
+                    "unpublishedPublications",
+                    "updatedAt",
+                    "variants",
+                    "vendor"
+                ]
+            }
+        },
+        "Shopify_ProductVariant": {
```

Comment:
Recently @robgrant brought up the fact that there are not webhooks for `ProductVariant`, only for `Product`. After discussing with @mcat we decided that we should drop `ProductVariant` as a shape that we want to highlight in this project. So remove this shape, its corresponding metadata shape and the related queries/mutations.

Reply:
Okay, yeah I was seeing the same thing. Will remove it now.